tests/driver/Firestore/InProcessFirestore.startAfter.test.ts

```diff
@@ -101,4 +102,32 @@ describe("In-process Firestore start after query", () => {
             { view: { name: "donkey" } },
         ])
     })
+
+    test("startAfter document id", async () => {
+        // Given there is a collection of documents with ids;
+        await db.doc("animals/22da618d").set({ name: "aardvark" })
+        await db.doc("animals/00a3382").set({ name: "badger" })
+        await db.doc("animals/11cbe6b5").set({ name: "camel" })
+
+        // When we order the collection by the document id;
+        const result = await db
+            .collection("animals")
+            .orderBy(FieldPath.documentId())
+            .startAfter("11cbe6b5")
+            .get()
+
+        // Then we should get the collection ordered by that field.
+        const docs: Array<{
+            id: number
+            name: string
+        }> = result.docs.map((doc) => doc.data() as any)
+
+        expect(docs.map((doc) => doc.name)).not.toStrictEqual([
```

Comment:
I don't see how this adds any value

Reply:
Merely keeping in style with the rest of the tests - I think it's been done to add legibility for the tester to understand the difference they should be seeing :)

Reply:
Have stripped it though as agree it doesn't particularly add much